en/setup/dev_env_linux_arch.md

```diff
@@ -1,27 +1,48 @@
-# Development Environment on ArchLinux
+# Development Environment on Arch Linux
 
-> **Note** These instructions allow you to build PX4 (without RTPS) for NuttX targets, using an unsupported version of GCCE from the package manager.
-The instructions have been tested on Antergos (an Arch Linux based distribution) as it is easier to set up than Arch Linux.
-We hope to provide fully tested instructions with the supported toolchain in the near future.
+> **Note** These instructions allow you to build PX4 (without RTPS) for NuttX targets and run simulation with jMAVsim or gazebo.
+The instructions have been tested on Manjaro (an Arch Linux based distribution) as it is much easier to set up than Arch Linux.
 
-## Permissions
+## Setup script
 
-The user needs to be added to the group "uucp":
+Using the the provided setup script it's very convenient to set your Arch installation up for PX4 development. Here are the steps to get you started from scratch.
 
+### Run the setup script
+The script by default installs all tools to build PX4 (without RTPS) for NuttX targets and run simulation with jMAVsim.
+
+Optionally you can specify the parameter `--gazebo` to also install gazebo simulation from the [AUR](https://aur.archlinux.org/packages/gazebo/). Note that gazebo gets compiled from source and therefore takes some time to install and requires entering the sudo password multiple times for dependencies.
```

Comment:
Would be good next time we update the script to match the ubuntu.sh approach - that has `--no-nuttx` and `--no-sim-tools` arguments.

Reply:
Yeah, it follows exactly that approach and has these same parameters, docs don't tell you 😬 
Gazebo not being enabled by defaults with "sim-tools" and having an additional parameter is done only because I consider it a bit less stable until I've seen how this AUR way of installing things holds up on the longer run. I'm kind of new to it. I can change it as soon as I have more insight into reproducibility.

Reply:
Added